splicemachine/spark/tests/NSDS test.ipynb

```diff
@@ -0,0 +1,1683 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Test spark"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "t1\n",
+      "1\n",
+      "after sc\n",
+      "after rdd\n",
+      "after collect\n",
+      "1\n",
+      "2\n",
+      "3\n",
+      "4\n"
+     ]
+    }
+   ],
+   "source": [
+    "import os\n",
+    "import inspect\n",
+    "# make sure pyspark tells workers to use python3 not 2 if both are installed\n",
+    "os.environ['PYSPARK_PYTHON'] = '/usr/bin/python3'\n",
+    "\n",
+    "\n",
+    "import pyspark\n",
+    "conf = pyspark.SparkConf()\n",
+    "print (\"t1\")\n",
+    "\n",
+    "print (1)\n",
+    "# create the context\n",
+    "sc = pyspark.SparkContext(conf=conf)\n",
+    "print(\"after sc\")\n",
+    "# do something to prove it works\n",
+    "rdd = sc.parallelize([1,2,3,4])\n",
+    "print(\"after rdd\")\n",
+    "\n",
+    "# collect the RDD to a list\n",
+    "list_elements = rdd.collect()\n",
+    "print(\"after collect\")\n",
+    "\n",
+    "# print the list\n",
+    "for element in list_elements:\n",
+    "    print(element)\n",
+    "count = 0"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Sql started successfully\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%sql\n",
+    "%defaultDatasource jdbc:splice://agent-000001:1527/splicedb;user=splice;password=kolj$6drornUrk"
```

Comment:
Are these credentials not an issue?  Ideally they would be passed in as a parameter.

Reply:
typically yes, and this isn't the final state of this notebook. This is just a testing suite I made for a cluster that doesn't exist anymore. Eventually I'll automate this and when I do I don't want recreate this code. Just leaving this file here in the interim